Cargo.lock

```diff
@@ -1098,11 +1057,23 @@ version = "0.7.3"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "6a6b1679d49b24bbfe0c803429aa1874472f50d9b363131f0e89fc356b544d03"
 dependencies = [
- "getrandom",
+ "getrandom 0.1.16",
+ "libc",
+ "rand_chacha 0.2.2",
+ "rand_core 0.5.1",
+ "rand_hc 0.2.0",
+]
+
+[[package]]
+name = "rand"
+version = "0.8.3"
```

Comment:
We should update our use of `rand` as well if we can, even though the second copy is just a build dependency.

Reply:
Unfortunately that pulls in a new version of `rand_core` which would require some updating of curve25519-dalek to avoid yet another duplicated dep.